src/llmcompressor/modifiers/awq/base.py

```diff
@@ -528,14 +520,26 @@ def smooth(module):
                     smooth(layer)
                 smooth(smooth_layer)
 
+            # remove caches needed to smooth this mapping
+            del self._smooth_activation_cache[mapping.smooth_name]
```

Comment:
Do we want to delete the parent kwargs cache?

Reply:
I moved that logic into `assert_all_activations_consumed`, which in hindsight is not great. I'll move it back to just below this line

Reply:
updated